src/main/js/generateMdAssets.js

```diff
@@ -51,42 +51,51 @@ export const genMdAssets = (doc, temp) => {
   })
 }
 
-export const genParamMove = (file, doc, intermediate, date) => {
+export const genParamMove = (contexts, intermediate, autoscope) => {
+  if (!autoscope) return contexts
+  return contexts.map((context) => {
+    const { file, data, date } = context
+
+    const validDate = new Date(date)
+    if (isNaN(validDate.getDate())) {
+      throw new Error(`${file} - invalid date`)
+    }
+    const findValue = intermediate.find((i) => {
+      return i.dir === path.dirname(file)
+    })
+    if (findValue && findValue.date < validDate) {
+      return { ...context, data: modifyData(data, findValue) }
+    }
+    const interData = {}
+    data.data.forEach((item) => {
+      interData[item.name.toLowerCase()] = item.ring.toLowerCase()
+    })
+    if (!findValue) {
+      intermediate.push({
+        dir: path.dirname(file),
+        data: interData,
+        date: validDate,
```

Comment:
this value is not a string. This may affect the static renderer.

Reply:
date string is in data.meta.date